### ISSUE TITLE
Attribute scroll down should have selected value visible when opened

### ISSUE THREAD

**Issue by umurb** [Cryptonomic/Arronax#145]
== Low Priority | Nice to have ==

1. Select an attribute somewhere from towards the end of the attributes list
2. Close the dropdown
3. Reopen the dropdown
4. Dropdown shows the top of the list

Expected: To see the value I selected and start scrolling from there.

![scroll-down anchor](https://user-images.githubusercontent.com/37525692/55344483-727cc180-547b-11e9-8379-b69b0cf26049.gif)

**Comment by Sm00g15** [comment]
Pending Umur's UI review/decision

**Comment by umurb** [comment]
No longer valid after #132
